config/build/webpack.config.vendor.js

```diff
@@ -0,0 +1,75 @@
+import mergeWith from 'lodash.mergewith';
+import webpack from 'webpack';
+import path from 'path';
+import { createBabelLoader, config } from './webpack.config.common.js';
+import { concat, appResolve, appModules } from '../utils';
+
+// eslint-disable-next-line
+const appDependencies = require(appResolve('package.json')).dependencies;
+const dependancies = {
```

Comment:
are we using american or british english ? in american, dependencies are spelled with an `e`, not an `a` :)